libs/ui/src/components/Explorer/_queries/useExplorerData.ts

```diff
@@ -57,9 +65,34 @@ export const useExplorerData = ({
     }>;
     pagination: null | {limit: number; offset: number};
     filters: IExplorerFilter[];
+    skip: boolean;
 }) => {
     const {lang: availableLangs} = useLang();
+
+    const queryFilters = filters
+        .filter(
+            ({value, condition}) =>
+                value !== null ||
+                [RecordFilterCondition.IS_EMPTY, RecordFilterCondition.IS_NOT_EMPTY].includes(condition)
+        )
+        .reduce<RecordFilterInput[]>((acc, {field, condition, value}, index) => {
+            if (index !== 0) {
+                acc.push({
+                    operator: RecordFilterOperator.AND
+                });
+            }
+
+            acc.push({
+                field,
+                condition,
+                value
+            });
+
+            return acc;
+        }, []);
+
```

Comment:
**note:** pour gérer les filtres multiples. On doit insérer un opérateur AND pour que ça soit valide coté back